commons/ava_testsuite/conflicting_txs_vertex_test/conflicting_txs_vertex_test_.go

```diff
@@ -0,0 +1,223 @@
+
+	byzantineClient, err := castedNetwork.GetGeckoClient(byzantineNodeServiceId)
+	if err != nil {
+		context.Fatal(stacktrace.Propagate(err, "Failed to get byzantine client."))
+	}
+
+	byzantineXChainAPI := byzantineClient.XChainApi()
+	// Three hardcoded X Chain Transactions
+	// Transaction1: Creates a new Fixed Cap Asset (no conflicts)
+	// Transaction2: Sends the UTXO to address1
+	// Transaction3: Sends the UTXO to address2
+	// Transactions 2 and 3 conflict, so this will trigger the byzantine node
+	// to issue all three transactions in a single vertex
+	// TODO move transaction strings into external test vector
+	createAssetTx := "111fqb8P5et4GYQHi6s3dyryAcmCqqPj998kfqvXtHvgnEmnoDwXBdHaYvjWp6WU1vhxGz3JTwBWXNWYBvJZkMb2jVoeTouJ6vjeQhQQx3MVYn2k5jYJGScd5bzrcE24AKLDG2YdCYRrfpJwgxPvHZH9XZuhzMy8Q8zZ1HzVEZggmRDysYUBBC"
+	conflictingTx1 := "111111yw4McR2ppKsF4t8AD8SmnkLVS4b9Zur5CikoDcC4dXr7rTYXfjc9bA45SiffbutqatRegMBRecAtCp55WXuGFGR1ymbJo5iCEFLbwLsbjaKVcYCyB5nyi6uwbHXdyz1cHVvnP9jDjVGT6dp3xzt57uXaXFGwxZky7ZSCL2Hh3vCuyjjZo7siGFMBzHmJc93SVTGptD6sJQoSiqqhdhnwLCTN6pKLYFfFYec2JMWSKo9jswtuY7JPWjEn8CNYzHxiBN3RN1MfbbLAwgFzAK321qpXUBaQjHq5vXj5GBqqkaW4UMhw2D5KPnSMzb4KPwussuT7YKJ4Rtmk7ysbD3sG4WbbL9kgQ2tzZFaLWa4vbEb51iUKDaZUuKZmdzcJxuk1nTwnbr3otKiEg"
+	conflictingTx2 := "111111yw4McR2ppKsF4t8AD8SmnkLVS4b9Zur5CikoDcC4dXr7rTYXfjc9bA45SiffbutqatRegMBRecAtCp55WXuGFGR1ymbJo5iCEFLbwLsbjaKVcYCyB5nyi6uwbHXdyz1cHVvnPGxtmVkdbkpw4wfLPpETgnvAWTJRUyrtWDTWtpHD19jArZqWfyZ6ipGiUpNVfU6yfaXah4CnmknXPR7hmm3jiWpoArUMJJd1e39vFCfZMDgov8MViCpUfwj6NpvaEyS7iWP5Ao6Lii6wq7VX5YJrRxhy9zFtgLCgLPya7ZdvwQfRSsQgRePoDXUmntJKfdPZoN4juG7t1ZdJ4KpPqvFbf2GhaUf2jzkJ7S6pTGAN9hthN71BV9CK9naHzQSJ9sZGQyZqPYdpt"
+
+	nonConflictId, err := byzantineXChainAPI.IssueTx(createAssetTx)
+	if err != nil {
+		context.Fatal(stacktrace.Propagate(err, "Failed to issue first transaction to byzantine node."))
+	}
+	conflictId1, err := byzantineXChainAPI.IssueTx(conflictingTx1)
+	if err != nil {
+		context.Fatal(stacktrace.Propagate(err, "Failed to issue second transaction to byzantine node."))
+	}
+	conflictId2, err := byzantineXChainAPI.IssueTx(conflictingTx2)
+	if err != nil {
+		context.Fatal(stacktrace.Propagate(err, "Failed to issue third transaction to byzantine node."))
+	}
+
+	logrus.Infof("Issued three transactions to Byzantine Node with IDs: %s, %s, %s", nonConflictId, conflictId1, conflictId2)
+
+	// Confirm the byzantine node Accepted the transactions
+	// Note: The byzantine behavior is to batch the pending transactions into a vertex as soon as it detects a conflict.
+	// It should try to accept each transaction before PushQuery-ing the vertex to other nodes to signal to this test
+	// controller that the vertex was successfully issued
+	status, err := byzantineXChainAPI.GetTxStatus(nonConflictId)
+	if err != nil {
+		context.Fatal(stacktrace.Propagate(err, fmt.Sprintf("Failed to get status of Transaction: %s", nonConflictId)))
+	} else if status != choices.Accepted.String() {
```

Comment:
Rather than making this an `else if`, can we make this a separate if statement, just to keep consistent with the pattern of "if condition then break"?

Reply:
Updating